documentation/modules/ROOT/pages/integrations/outbox.adoc

```diff
@@ -133,35 +133,90 @@ The table name to be used when creating the outbox table.
 |OutboxEvent
 
 
-|`quarkus.debezium-outbox.id-column-name`::
+|`quarkus.debezium-outbox.id.name`::
 The column name for the event id column.
 |string
 |`id`
 
-|`quarkus.debezium-outbox.aggregate-id-column.name`::
+|`quarkus.debezium-outbox.id.column-definition`::
+The database-specific column definition for the event id column.
+|string
```

Comment:
Is column definition really "string"? Isn't that rather the _property_ type, whereas column definition would be something like "VARCHAR(2000)" or "JSONB"? Is there an example somewhere that shows these values being set?

Reply:
That's correct, the documentation here followed that of how Quarkus lists their properties with the property name, its type, and its default value.  Should we make this like our other property tables?